### PR TITLE
Refactor user-microservice

### PR DESCRIPTION
**Pull request by chiahuang**

### CODE REVIEW

File: userapp/.env [hunk 1]
```diff
@@ -0,0 +1 @@
+PORT=3003
```
Comment (i-mckinney) [inline userapp/.env:1]: is there a reason why this isn't ignored?

File: userapp/src/components/controls/HelixTextField.js [hunk 2]
```diff
@@ -8,40 +8,29 @@ const HelixTextField = (props) => {
...
-      type = "", 
-      error = false, 
       label, 
-      InputLabelProps = {}, 
       value,
-      required = false,
-      helperText = "", 
-      placeholder = "", 
       onChange, 
+<<<<<<< HEAD
```
Comment (i-mckinney) [inline userapp/src/components/controls/HelixTextField.js:21]: clean up the merge conflicts here

File: userapp/.env (hunk 1 above)
Comment (chiahuang) [reply userapp/.env:1]: I did not add .env into gitignore. Added!

File: userapp/src/components/controls/HelixTextField.js (hunk 2 above)
Comment (chiahuang) [reply userapp/src/components/controls/HelixTextField.js]: Cleaned up!

**Pull request merged**